### ISSUE TITLE
Chrome Apps deprecation

### ISSUE THREAD

**Issue by mems** [kzahel/web-server-chrome#221]
Chrome Apps [are now deprecated](https://blog.chromium.org/2020/01/moving-forward-from-chrome-apps.html) and it support will be removed from Chrome.
It look like [chrome.sockets.tcpServer](https://developer.chrome.com/apps/sockets_tcpServer) [don't have any fallback](https://developers.chrome.com/apps/migration) for web platform or extension migration.

What do you suggest as replacement for the Chrome App Web Server for Chrome ? Look like migrate to WebExtension is not an viable solution (unless you use [native Messaging](https://developer.chrome.com/extensions/nativeMessaging))

**Comment by kzahel** [comment]
Yes there aren't enough permissions to implement as an extension unfortunately.
Not sure what we'll do, I guess just let it slowly go away

**Comment by ethanaobrien** [comment]
Should we use an external app as described in #116

**Comment by kzahel** [comment]
That would be an option, certainly. I always assumed if somebody can install an electron app, they might as well just open their terminal and type `python3 -m http.server`, but I can see how having an easy installer would be good for users who just want to click and not go to the console.

**Comment by kzahel** [comment]
The original Chrome App is no longer supported. The app has been rebuilt from scratch — see https://ok200.app for the new versions and to sign up for updates.
